src/styles/components/_forms.scss

```diff
@@ -0,0 +1,87 @@
+@import '../variables';
+
+.form-control {
+
+  &, &:focus {
+    color: $dark;
+    color: var(--dark);
+  }
+
+  font-size: 1.4rem;
+  line-height: 1.6rem;
+  font-weight: 400;
+  height: auto;
+  border-radius: $input-border-radius;
+  padding: $input-padding-y $input-padding-x;
+  background-color: $input-background-color;
+  border-color: $input-border-color;
+
+  &:focus {
+    border-color: $info;
+    background-color: $light-01;
+    box-shadow: none;
+  }
+
+  // Override browser autocomplete background color
+  &:-webkit-autofill {
+    -webkit-box-shadow: 0 0 0 50px white inset;
+    box-shadow: 0 0 0 50px white inset;
+  }
+
+  &:-ms-input-placeholder,
+  &::placeholder { /* Chrome, Firefox, Opera, Safari 10.1+ */
+    color: $dark-01;
+    opacity: 1; /* Firefox */
+  }
+  
+  // &:-ms-input-placeholder { /* Internet Explorer 10-11 & Edge */
+  //   color: $dark-01;
```

Comment:
Can probebly remove this for now, since we don't support anything with IE10 and we have the same rule above

Reply:
Okay, tested this, unfortunately need to keep it separated to make it work :(